docs/Schema-changes.md

```diff
@@ -0,0 +1,164 @@
+
+#### Credential title
+The credential title is the main attribute you want users to see when viewing a credential. An example of this would be the business number of a business number credential. See figure 1 for an example.
+
+![](./assets/schema_label_cred_title.png)  
+*Figure 1: Schema label and credential title*
+  
+#### Highlighted attributes
+Highlighted attributes are secondary credential attributes that you believe would be important to a user viewing a specific credential. An example of this would be a Cannabis marketing license credential. A user would likely be interested in knowing the effective and expiry date of the credential, but probably would not care much about the issue date since it does not have anything to do with the validity of the credential. Aries-vcr allows issuers to include only relevant credential attributes in their credential displays. See figure 2 for an example.  
+
+![](./assets/highlighted_attributes.png)  
+*Figure 2: Highlighted attributes*
+
+#### Schema labels
+Aries-vcr has added the functionality to specify the name of a credential schema in a multilingual format. These schema names appear as the title of the credential and let users know what type of credential they are viewing. This schema label is optional and is only needed if you plan on supporting multi language use. If no schema label is defined, a good fallback is to use the credential type description displayed in the same way as a schema label. The new version of Orgbook takes this approach. See figure 1 for example of a schema label in Orgbook.
+  
+#### Aries-vcr defaults
+##### Credential title:
+If the credential title is not specified in a credential schema, or there is no claim label to match the credential title, then applications such as Orgbook may instead use the registration number of the entity holding the credential. See figure 3 for an example  
+![](./assets/default_cred_title.png)  
+*Figure 3: default credential title and highlighted attributes*
+  
+##### Highlighted attributes:
+If no highlighted attributes are specified, or there are no claim labels to match the highlighted attributes, then applications like Orgbook will not display any highlighted attributes. See figure 3.1 to see how a credential with no highlighted attributes or credential title is displayed.  
+![](./assets/default_highlighted.png)  
+*Figure 3.1: Default credential title and highlighted attributes*
+
+#### Recommended changes to schema
+To incorporate these optional changes, Aries-vcr recommends reviewing the new schema attributes and how to add them to your credential schema in the section below.
+
+~~~
+{
+  "schema_name": "registration.registries.ca",
+  "schema_version": "1.0.44",
+  "attributes": [
+    "registration_id",
+    "registration_date",
+    ...
+    "reason_description",
+    "expiry_date"
+  ],
+  "metadata": {
+    "topic": [
```

Comment:
This message structure doesn't match the structure passed to the `/registration` endpoint nor the structure of the message passed from one agent to the other as part of the registration protocol.

Is this the message structure internal to the new Issuer Agency?